lighthouse-core/lib/page-functions.js

```diff
@@ -438,40 +448,45 @@ function getBoundingClientRect(element) {
   };
 }
 
-/*
+/**
  * RequestIdleCallback shim that calculates the remaining deadline time in order to avoid a potential lighthouse
  * penalty for tests run with simulated throttling. Reduces the deadline time to (50 - safetyAllowance) / cpuSlowdownMultiplier to
  * ensure a long task is very unlikely if using the API correctly.
  * @param {number} cpuSlowdownMultiplier
- * @return {null}
+ * @return {void}
```

Comment:
can we just remove the return keyword at the end of this fn, and remove the return jsdoc?

Reply:
> can we just remove the return keyword at the end of this fn, and remove the return jsdoc?

Sure. I'm not sure if that `null` was from a previous incarnation of the function or something, but no need to keep it

Reply:
I'm curious why :)

Seems like a good thing to avoid accidentally relying on an implicit return type of some sort

Reply:
I favor consistency, even if there are potential benefits. If there are benefits we want, we should do this everywhere, and in a big PR :)

Reply:
> Seems like a good thing to avoid accidentally relying on an implicit return type of some sort, not that I think I'd ever explicitly suggest doing it on a proactive basis, but I wouldn't think it hurts anything.

oh, for functions with an implicit return we usually leave it blank (instead of an explicit `return {void}`). Would you prefer it the other way? I do find inferred return types kind of dangerous in general, so I could be convinced...

(@connorjclark was mistaken that there was a return keyword to remove...the returns are nested one level deep in the `requestIdleCallback` replacement functions)

Reply:
I agree on the consistency bit, just only to a point. I have no strong opinions, just was a little surprised that one would about implicit void return types :) I guess it's just a case of different thresholds then, carry on!